src/abe_policy/partitions.rs

```diff
@@ -63,17 +57,17 @@ impl From<&[u8]> for Partition {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use cosmian_crypto_core::bytes_ser_de::Deserializer;
 
     #[test]
     fn test_partitions() -> Result<(), Error> {
         let mut values: Vec<u32> = vec![12, 0, u32::MAX, 1];
         let partition = Partition::from_attribute_values(values.clone())?;
-        let bytes = partition.0;
-        let mut readable = &bytes[..];
         // values are sorted n Partition
         values.sort_unstable();
+        let mut de = Deserializer::new(&partition);
         for v in values {
-            let val = leb128::read::unsigned(&mut readable).expect("Should read number") as u32;
+            let val = de.read_u64().unwrap() as u32;
```

Comment:
I could change this into `read_leb128_u64()` or `read_leb128_uint()`? This would be done in `CryptoCore`. I could do it in [this](https://github.com/Cosmian/crypto_core/pull/36) PR.

Reply:
done here be522da